examples/hotrod/pkg/tracing/init.go

```diff
@@ -30,14 +30,18 @@ import (
 
 // Init creates a new instance of Jaeger tracer.
 func Init(serviceName string, metricsFactory metrics.Factory, logger log.Factory) opentracing.Tracer {
-	cfg, err := config.FromEnv()
-	if err != nil {
-		logger.Bg().Fatal("cannot parse Jaeger env vars", zap.Error(err))
+	cfg := &config.Configuration{
```

Comment:
It might become clearer later, but why is this change needed?

Reply:
This makes hotrod respect env vars which I used in `/docker-compose/jaeger-docker-compose.yml` to setup hotrod as a local test of adaptive sampling.

Without this change hotrod is hardcoded at `const`/`1`.

Reply:
Right, I see. Now, it is const/1, but overridden from the env var if values are provided there. Sounds good!